src/GravityFormsExtensions.php

```diff
@@ -322,6 +322,8 @@ public function p4_gf_custom_confirmation($confirmation, $form)
                             "gGoal":  "' . ($form['p4_gf_type'] ?? self::DEFAULT_GF_TYPE) . '",
                             "formTitle": "' . $form['title'] . '"
                         });
+                        // Disable GFTrackEvent (GFTrackEvent is belongs to Gravity Forms Google Analytics Add-On)
```

Comment:
Small typo here in the comment, we should remove `is` from `is belongs to` 🙂

Reply:
Fixed! 🙂